### PR TITLE
Update the remaining tests to use Enzyme

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
[Update remaining tests to use Enzyme](https://trello.com/c/a2iQ7MeR/454-update-remaining-tests-to-use-enzyme)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8604/

File: jujugui/static/gui/src/app/components/terminal/terminal.js [hunk 1]
```diff
@@ -45,9 +44,7 @@ class Terminal extends React.Component {
     const term = new XTerm();
     term.writeln('connecting (this operation could take a minute)... ');
     this.term = term;
-    term.open(
-      ReactDOM.findDOMNode(this).querySelector('.juju-shell__terminal'),
-      true);
+    term.open(this.refs.terminal, true);
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/terminal/terminal.js]: Nice.

**Comment by frankban** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**